### ISSUE TITLE
Logic network bug, (Jotunn - Self-conscious terminal commands don't return same output as before)

### ISSUE THREAD

**Issue by JoneKone** [Regalis11/Barotrauma#8403]
**Description**
A ship that I use doesn't work properly on the new update. Logic error/bug

**Steps To Reproduce**
Get ship
```
https://steamcommunity.com/sharedfiles/filedetails/?id=2183071144&searchtext=jotunn
```
Type help on terminal (Sub editor)
Try on 0.15.x.x version works. "Prints all help text properly"
Try on 0.16.7.x version doesn't work. "Prints only 1 line of help text"

**Version**
0.16.7.0

**Additional information**
Logic error of some kind, I have no idea. what to call it.

**Comment by Regalis11** [comment]
I attempted to debug this, but due to the complexity of the circuit (and my lack of understanding on how it's intended to work) I wasn't able to get to the bottom of the issue. I'm sorry to say but I think we're going to need a simpler test case to be able to diagnose this.

**Comment by JoneKone** [comment]
Although the author of the ship fixed the issue (and has uploaded an updated version of the ship to workspace).. I feel like something in the logistics code, either a quark has been fixed or something was broken, in the 0.16.7.x.x. I'm posting this comment very hesitantly, as I have no way of confirming it either way. Also as a non issue now due to the fact that it was fixed in the ship design.
```
https://steamcommunity.com/sharedfiles/filedetails/?id=2182762687
```
the author said in he's comments.
```
I have now updated the AI to be much more stable when using the "help" command. Let me know if you still find problems or any other problem.
```
